### ISSUE TITLE
ClojureBridge contact

### ISSUE THREAD

**Issue by plexus** [clojurebridge-berlin/organization#52]
- [x] Submit for the ClojureBridge.org website
- [x] Add sponsors as we find them
- [ ] Send announcement of the event to the clojurebridge-workshops ML
- [x] Send summary/report after the event to the clojurebridge-wokshops ML
- [ ] Get suggestions/improvements for coaches training "upstream"
- [ ] Get suggestions/improvements for curriculum "upstream"

**Comment by plexus** [comment]
Submitted for putting up the event page:

https://github.com/ClojureBridge/Workshops/issues/26

**Comment by plexus** [comment]
We already did a bit better this time at communicating back. I wrote to the global mailing list about some of the more interesting aspects of our way of working, and shared the blog post which gives a good general recap of the day. With Malwine joining the board, and me being there as backup, we should start getting more of a dialogue between our group and the wider ClojureBridge community.

Contributing curriculum and coaches training slides back up is something we can try to work on for the next workshop.
